### ISSUE TITLE
How to delete duplicate entries?

### ISSUE THREAD

**Issue by Serif-7** [jrnl-org/jrnl#1294]
Somehow, several entries in my jrnl have been duplicated, so now there are three of each entry, all write next to each other with the same timestamp.

Is there a simple way to select and delete duplicate entries via `--delete` or some other method like `sed`?

**Comment by Serif-7** [comment]
`awk '!seen[$0]++' file.txt > file_new.txt` works well I've found, though it deletes the blank lines between entries. For fixing that that I've found 
`sed "s#\[#\n[#g" file.txt > file_spaced.txt` works.

**Comment by wren** [comment]
Although we don't have an explicit command to find dupes (maybe we should?), you can use jrnl's import command for this.

1. Move your journal file (let's say `jrnl.txt`) to something else (like `jrnl-backup.txt` or anything you want)
2. Import the backup into the (now empty) journal: `cat jrnl-backup.txt | jrnl --import`
3. jrnl will recreate `jrnl.txt` and dedupe the entries as they're being imported

Afterward, you can diff `jrnl.txt` and `jrnl-backup.txt` to be sure you got what you expected.

**Comment by Serif-7** [comment]
@wren alright, I'm running into weird behavior now. I... don't know when this happened, but I checked my journal and some entries were duplicated, even after I explicitly removed them using the commands I listed. Then I tried your methods, and it did remove some dupes, but the ones that were duplicated still had a single duplicated entry alongside them.

This one might require some investigation on my part.
